src/main/kotlin/it/pagopa/wallet/services/WalletService.kt

```diff
@@ -61,6 +64,44 @@ class WalletService(@Autowired private val walletRepository: WalletRepository) {
             .map { LoggedAction(it, WalletPatchEvent(it.id.value.toString())) }
     }
 
+    fun findWallet(walletId: UUID): Mono<WalletInfoDto> {
```

Comment:
this method returns a wallet given an id but I don't see useId, so it is required to check the user who is requesting that wallet

Reply:
Here we have the findByWalletId. We have also findByUserId but since userId is not a key, it may return more than one wallet. Do we have to update getWalletById signature adding the userId or do we have to use the getWalletByUserId ?